org.worldpossible.ScriptLauncher.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.worldpossible.ScriptLauncher
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+branch: eos3
+sdk: org.gnome.Sdk
+command: worldpossible-scriptlauncher
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # We need to be able to run commands on the host to execute scripts chosen by the user
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
+
+build-options:
+  env: { V: '1' }
+
+cleanup:
+  - /include
+  - /man
+  - /share/man
+  - "*.la"
+  - "*.a"
+
+modules:
+  - name: worldpossible-scriptlauncher
+    buildsystem: meson
+    builddir: true
+    sources:
+      - type: git
+        branch: 2b0212f717e20c2f4bfcb9f7c4958e0614d6e1cc
```

Comment:
Does this work?

```suggestion
        commit: 2b0212f717e20c2f4bfcb9f7c4958e0614d6e1cc
```

Reply:
I did it this way based on https://github.com/flathub/org.gnome.Builder/blob/master/org.gnome.Builder.json#L645